SimpleSpotify/color.ini

```diff
@@ -0,0 +1,20 @@
+[Base]
+main_fg                               = 999999
+secondary_fg                          = d8d8d8
+main_bg                               = 1b1b1b
+
+sidebar_and_player_bg                 = 1b1b1b
+slider_bg                             = 1b1b1b
+sidebar_indicator_and_hover_button_bg = 1fc167
+indicator_fg_and_button_bg            = 1fc167
+
+cover_overlay_and_shadow              = 222
+
+pressing_fg                           = 999999
+scrollbar_fg_and_selected_row_bg      = yellow
+pressing_button_fg                    = green
+pressing_button_bg                    = 111
+selected_button                       = brown
+miscellaneous_bg                      = 111
+miscellaneous_hover_bg                = purple
+preserve_1                            = pink
```

Comment:
You probably left this by mistake, as well as the empty README in the same folder